x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/security/authz/RoleDescriptor.java

```diff
@@ -600,6 +613,11 @@ public IndicesPrivileges(StreamInput in) throws IOException {
             this.deniedFields = in.readOptionalStringArray();
             this.privileges = in.readStringArray();
             this.query = in.readOptionalBytesReference();
+            if (in.getVersion().onOrAfter(Version.V_6_7_0)) {
```

Comment:
This is right for the end-goal when we backport this to 6.7, but it will cause BWC to fail when it is merged to `master` as 6.7 won't be including the boolean field in the stream yet.
We typically commit it as `V_7_0_0` and then update it to `V6_7_0` at time of backport.